_opensearch/snapshots/sm-api.md

```diff
@@ -0,0 +1,460 @@
+          "timezone" : "UTC"
+        }
+      },
+      "time_limit" : "1h"
+    },
+    "deletion" : {
+      "schedule" : {
+        "cron" : {
+          "expression" : "0 1 * * *",
+          "timezone" : "America/Los_Angeles"
+        }
+      },
+      "condition" : {
+        "max_age" : "7d",
+        "min_count" : 7,
+        "max_count" : 21
+      },
+      "time_limit" : "1h"
+    },
+    "snapshot_config" : {
+      "indices" : "*",
+      "metadata" : {
+        "any_key" : "any_value"
+      },
+      "ignore_unavailable" : "true",
+      "timezone" : "America/Los_Angeles",
+      "include_global_state" : "false",
+      "date_format" : "yyyy-MM-dd-HH:mm",
+      "repository" : "s3-repo",
+      "partial" : "true"
+    },
+    "schedule" : {
+      "interval" : {
+        "start_time" : 1656425122909,
+        "period" : 1,
+        "unit" : "Minutes"
+      }
+    },
+    "enabled" : true,
+    "last_updated_time" : 1656425122909,
+    "enabled_time" : 1656425122909,
+    "notification" : {
+      "channel" : {
+        "id" : "NC3OpoEBzEoHMX183R3f"
+      },
+      "conditions" : {
+        "creation" : true,
+        "deletion" : false,
+        "failure" : false,
+        "time_limit_exceeded" : false
+      }
+    }
+  }
+}
+```
+
+### Parameters
+
+You can specify the following parameters to create/update an SM policy.
+
+Parameter | Type | Description 
+:--- | :--- |:--- |:--- |
+`description` | String | The description of the SM policy. Optional.
+`enabled` | Boolean | Should this SM policy be enabled at creation. Optional.
+`snapshot_config` | Object | The configuration options for snapshot creation. Required.
+`snapshot_config`<br>&nbsp;&nbsp;&nbsp;&nbsp;`date_format` | String | Snapshot names have the format `<policy_name>-<date>-<random number>`. `date_format` specifies the format for the date in the snapshot name. Supports all date formats OpenSearch supports. Optional. Default is "yyyy-MM-dd'T'HH:mm:ss".
+`snapshot_config`<br>&nbsp;&nbsp;&nbsp;&nbsp;`date_format_timezone` | String | Snapshot names have the format `<policy_name>-<date>-<random number>`. `date_format_timezone` specifies the time zone for the date in the snapshot name. Optional. Default is UTC.
+`snapshot_config`<br>&nbsp;&nbsp;&nbsp;&nbsp;`indices` | String | The names of the indices in the snapshot. Multiple index names are separated by `,`. Supports wildcards (`*`). Optional. Default is `*` (all indices).
+`snapshot_config`<br>&nbsp;&nbsp;&nbsp;&nbsp;`repository` | String | The repository to store snapshots. Required.
+`snapshot_config`<br>&nbsp;&nbsp;&nbsp;&nbsp;`ignore_unavailable` | Boolean | Ignore unavailable indices? Optional. Default is `false`.
+`snapshot_config`<br>&nbsp;&nbsp;&nbsp;&nbsp;`include_global_state` | Boolean | Include cluster state? Optional. Default is `true` because of [Security Plugin considerations]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/#security-plugin-considerations).
```

Comment:
Unless the link reflects the title of the page, "plugin" should be lowercase.

Reply:
Done